cpp/cmake/config.json

```diff
@@ -12,6 +12,14 @@
           "FIND_PACKAGE_ARGUMENTS": "*" 
         }
       },
+      "rapids_cython_create_modules": {
```

Comment:
Teaching `cmake-format` the signature of this function helps it to split it up into one keyword argument per line.